### PR TITLE
To do list add remove

### PR DESCRIPTION
**Pull request by vikas924**
Dear Reviewer,

In this PR we have implemented the required below features in the To do list structure exercise.

Create html for to do list structure.

Add styles for to do list structure.

Create a function for display tasks dynamically.

Added function for add task.

Added function for remove task.

Added function for edit task.

Use webpack for bundle js and show on webpack dev server.

### CODE REVIEW

File: src/modules/edit.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    const index = Array.from(label).indexOf(event.target);
+    label[index].innerHTML = inputvalue;
+    tasks[index].description = label[index].innerHTML;
+    localStorage.setItem('array', JSON.stringify(tasks));
+    label[index].blur();
+    const div = document.querySelectorAll('#style');
+    div[index].style.backgroundColor = '#fff';
+    this.style.backgroundColor = '#fff';
+  }
+}
```
Comment (VICTOR-CHUKWU) [inline src/modules/edit.js]: - [x] when I edit and click outside, the description of the edited item becomes an empty string in the local storage, kindly fix this
Screenshot from 2023-03-30 10-31-01
look closely to the second item in the local storage of the image above and see that the description is an empty string

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @vikas924  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/edit.js (hunk 1 above)
Comment (vikas924) [reply src/modules/edit.js]: i did not had any issues in firefox. i fixed it now switched to .value rather than .innerhtml

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @vikas924,

✔️ Edit functionality works properly
✔️ Index values are updated upon deleting a task

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**